MovingBlocks/Terasology
# Fix for background music not looping

### Contains

Fixes #2527 
### How to test

Start Game wait for main menu track to finish and start again. Approximately 3:31 seconds.
Start Game music should stop
Exit to menu background music should start again
### Outstanding before merging
